SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -126,6 +126,8 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bUseSecureServerConnection(false)
 	, bEnableClientQueriesOnServer(false)
 	, StartupLogRate(5.0f)
+	, bEventTracingEnabled(false)
+	, MaxEventTracingFileSizeBytes(256 * 1024 * 1024) // 256mb
```

Comment:
probably put this number as a constexpr in an anon namespace. Even commented it's better ot not have the magic numbers.